sound/soc/sof/intel/telemetry.c

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
+// This file is provided under a dual BSD/GPLv2 license.  When using or
+// redistributing this file, you may do so under either license.
+//
+// Copyright(c) 2023 Intel Corporation. All rights reserved.
+//
+//
```

Comment:
nitpick: do you need these empty comment lines?

Reply:
thanks, removed